.github/workflows/test.yml

```
@@ -0,0 +1,37 @@
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest pytest-cov
+          pip install -e .
+
+      - name: Run tests
+        run: pytest test/ -v --cov=hivemind_presence --cov-report=term-missing
+
+      - name: Upload coverage
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.11)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)